File: Php/app/Urls/HomegameDetailsUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class HomegameDetailsUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::homegameDetails, $homegame);
		}

	}

}

File: Web/Models/HomegameDetailsUrlModel.cs
using Core.Classes;
using Web.Models;
using Web.Routing;

namespace app{

	public class HomegameDetailsUrlModel : HomegameUrlModel{

		public HomegameDetailsUrlModel(Homegame homegame) : base(RouteFormats.HomegameDetails, homegame){}

	}

}